v2/anomalydetection/backend-setup.mdx

```diff
@@ -0,0 +1,229 @@
+---
+sidebar_position: 3
+title: Backend Setup
+hide_title: true
+---
+
+# Setting up the Backend
+
+In order to use the anomaly detection service, a request to the following enpoint (based on the region) needs to be made:
+
+**US Region:**
+```typescript
+POST https://security-us-east-1.aws.supertokens.io/v1/security
+```
+
+**EU Region (Ireland):**
+```typescript
+POST https://security-eu-west-1.aws.supertokens.io/v1/security
+```
+
+**APAC Region (Singapore):**
+```typescript
+POST https://security-ap-southeast-1.aws.supertokens.io/v1/security
+```
+
+You can view the HTTP API reference for this endpoint [here](#somelinkhere).
```

Comment:
You can make this a collapsible element and then add the full json in this page itself?

Reply:
done. i think we need to have a more dynamic version, as this doesn't support stuff like brute force response which has the "key" key optional.